src/coredata.cs

```diff
@@ -127,6 +137,9 @@ public interface NSEntityDescription : NSCoding, NSCopying {
 		[Export ("managedObjectClassName")]
 		string ManagedObjectClassName { get; set; }
 
+		[Export ("renamingIdentifier")]
+		string RenamingIdentifier { get; set; }
```

Comment:
Good point, in the tests I have the following and passes:

`
[Test]
public void GetSetRenamingIdentifier ()
{
	using (var pd = new NSPropertyDescription ()) {
		Assert.IsNull (pd.RenamingIdentifier);
		pd.RenamingIdentifier = "Foo";
		Assert.AreEqual ("Foo", pd.RenamingIdentifier);
	}
}
`

So AFAIK we can use a managed string.